app/src/main/java/com/alexandr4/loftcoin/screens/rate/CurrencyDialog.java

```diff
@@ -0,0 +1,70 @@
+package com.alexandr4.loftcoin.screens.rate;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+
+import com.alexandr4.loftcoin.R;
+import com.alexandr4.loftcoin.utils.Fiat;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.DialogFragment;
+
+public class CurrencyDialog extends DialogFragment {
+
+    public static final String TAG = "CurrencyDialog";
```

Comment:
where do you use this?

Reply:
it is used in RateFragment by a call of a window of dialogue of the choice of currency. for identification of a window of dialogue.